compose/vtorc-up.sh

```diff
@@ -36,6 +39,14 @@ else
     cp /vt/orchestrator/default.json /vt/orchestrator/config.json
 fi
 
+sleep $sleeptime
+
+# set the correct durability policy for the keyspace
+for keyspace in $keyspaces; do
+  echo "Set Keyspace Durability Policy ${keyspace} to none"
+  /vt/bin/vtctldclient --server vtctld:$GRPC_PORT SetKeyspaceDurabilityPolicy --durability-policy=none $keyspace
```

Comment:
I confirmed that executing this command won't trigger vtorc after vtorc decided to ignore vttablets.
Which means it's safer to set the durability policy before launching vtorc.

Reply:
[memo] By the way, once `vtorc` ignores vttablets, running PlannedReparentShard after SetKeyspaceDurabilityPolicy wakes up `vtorc`.